_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,299 @@
+  might not be able to start using that ability until a transaction is
+  broadcast, and they may lose the ability to steal funds controlled by
+  that key once the transaction is confirmed to a certain depth.  This
+  limits the amount of time available for their attack and means a slow
+  attack might not work.  Although this has previously been discussed at
+  length in the context of taproot's choice to directly use public keys
+  in the P2PK style (see [1][tap qc1], [2][tap qc2], and newsletters
+  [#70][news70 qc] and [#86][news86 qc]), it was the subject of [renewed
+  discussion][dashjr quantum] this week on the Bitcoin-Dev mailing list
+  after the publication of an [email][friedenbach post] opposing taproot
+  out of fear that we could see a quantum computer powerful enough to
+  attack Bitcoin-style public keys "as soon as the end of the decade."
+
+  None of the participants in the mailing list discussion said they also
+  opposed taproot, but they did examine the argument's premises, discuss
+  alternatives, and evaluate what tradeoffs would be implied by those
+  alternatives.  A selection of those conversations are summarized below:
+
+    - *Hashes not currently doing a good job at QC resistance:*
+      as of a [2019 survey][wuille stealable], an attacker with a
+      powerful QC and nothing else besides a copy of the Bitcoin block
+      chain could steal over 1/3 of all bitcoins.  Most of those would
+      be the result of users [reusing addresses][topic output linking],
+      a discouraged practice---but one that seems unlikely to go away
+      soon.
+
+        Additionally, discussion participants pointed out that anyone
+        who shares their individual public keys or [BIP32][] extended
+        public keys (xpubs) with third parties would also be at risk
+        from a powerful QC if their public keys leaked.  This would
+        likely include most bitcoins stored with a hardware wallet or in
+        an LN payment channel.  In short, it's possible that even though
+        we almost universally use P2PKH-style hashed public keys today,
+        nearly all bitcoins could be stolen by a powerful QC with access
+        to public or third-party data.  That implies that the choice to
+        use P2PK-style non-hashed public keys with taproot doesn't
+        significantly change Bitcoin's current security model.
+
+    - *Taproot improvement in post-QC recovery at no onchain cost:*
+      if Bitcoiners learn that a powerful QC computer has manifested, or
+      soon will, they can reject any taproot key-path spends---the type
+      of spend where only a single signature is used.  However, even
+      people who exclusively expect to use key-path spends can create
+      taproot addresses that are ready to use script-path spends which
+      commit to a hash which itself commits to at least one
+      taproot script ([tapscript][topic tapscript]).  That hash
```

Comment:
"However, even people [...] one". I found this sentence quite long and difficult to parse. Is it possible to break it up a bit?